src/pages/portfolio/Positions.tsx

```diff
@@ -21,8 +24,8 @@ export const Positions = () => {
   const navigate = useNavigate();
 
   const shouldRenderTriggers = useSelector(calculateShouldRenderTriggersInPositionsTable);
-  // TODO: CT-503
-  // const shouldRenderActions = useSelector(calculateShouldRenderActionsInPositionsTable);
+  const shouldRenderActions = useSelector(calculateShouldRenderActionsInPositionsTable);
+  const showClosePositionAction = false;
```

Comment:
is it possible to return `shouldRenderActions` as a `bool` instead of a `fn`?

`const shouldRenderActions = useSelector(calculateShouldRenderActionsInPositionsTable(shouldClosePositionAction)`